src/features/collections/CollectionDetail.tsx

```diff
@@ -216,8 +226,12 @@ export const CollectionDetail = () => {
           <FilterMenu
             collectionId={collection.id}
             anchorEl={filterMenuRef.current}
-            open={filterOpen}
-            onClose={() => setFiltersOpen(false)}
+            open={filterPanelOpen ?? false}
+            onClose={() => {
+              if (collection?.id) {
+                dispatch(setFilterPanelOpen([collection.id, !filterPanelOpen]));
```

Comment:
Probably don't need to pass the collectionId, can just put filterPanelOpen in the collectionSlice top level state